packages/canary-docs/docs.json

```diff
@@ -5099,7 +4971,7 @@
       "path": "src/components/ic-data-table/ic-data-table.types.tsx"
     },
     "src/components/ic-data-table/ic-data-table.types.tsx::IcDataTableDataType": {
-      "declaration": "{ [key: string]: any }",
+      "declaration": "{\n  // eslint-disable-next-line @typescript-eslint/no-explicit-any\n  [key: string]: any;\n  rowOptions?: IcDataTableRowOptions;\n}",
```

Comment:
Sorry just noticed that the ESLint disable comment from ic-data-table.types.tsx is getting added to this docs.json file - I think it will need addressing somehow as this will show up on the Types page on the website